config_test.go

```diff
@@ -65,3 +65,79 @@ func TestConfig_AuthMechanism(t *testing.T) {
 		})
 	}
 }
+
+func TestConfig_Nodes(t *testing.T) {
+	testCases := []struct {
+		name    string
+		config  DestinationConfig
+		wantErr bool
+	}{{
+		name: "port is greater than 65535",
+		config: DestinationConfig{
+			Nodes: []string{
+				"127.0.0.1:99999",
```

Comment:
Can we add a test for a hostname without a port? That's also allowed, right?

Reply:
it's allowed by gocql yes, but wasn't sure if I should allow it to the user (lots of options and would have a longer description), but now that I'm thinking more about it, I think we should allow it yes.
I'm gonna have to do some validation changes to allow it first, then I'll add a test.